css/properties/border-inline-start-style.json

```diff
@@ -0,0 +1,78 @@
+{
+  "css": {
+    "properties": {
+      "border-inline-start-style": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-inline-start-style",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "firefox": [
+              {
+                "version_added": "41"
+              },
+              {
+                "version_added": "38",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.vertical-text.enabled",
+                  "value_to_set": "true"
+                },
+                "notes": "Enabled by default since Firefox 41."
+              },
+              {
+                "version_added": "38",
+                "alternative_name": "-moz-border-start-style"
```

Comment:
Similar to `-moz-border-start-color`, the original table noted `-moz-border-end-color` here. I assumed `-moz-border-start-style` was intended and corrected it here.